### DISCUSSION TITLE
Confusing example sentence for 盗っ人 / 盗人 entry

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#30] [Errors and Corrections]
Japanese: 盗人にも三分の理

English: Give the devil his due.

---

Even ignoring the translation of the proverb/phrasing which I admit I'm not very familiar with (either in English or Japanese) but I feel like it's a mistranslation... the actual example sentence itself is very confusing and not very indicative of how the word 盗っ人/盗人 works (I include both っ and non-っ variants as it shows up in both due to being the same entry/alternative spelling).

I don't think this example sentence is very useful, unfortunately.

**Comment by stephenmk** [comment]
Proverbs that require supplemental explanations don't make for great example sentences, I agree. We just removed this one. Thanks.
